src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -61,9 +58,54 @@ public IActionResult CreateUsers([FromBody] UserCreateDTO userDto)
             this._context.SaveChanges();
 
             // returns a url of the user that was created 
+            // https://www.code4it.dev/blog/createdAtRoute-createdAtAction
             // https://stackoverflow.com/questions/47939945/how-to-use-created-or-createdataction-createdatroute-in-an-asp-net-core-api#64315534
             return CreatedAtAction(nameof(GetUser),new {id = userDto.Id },  userDto);
         } 
 
+        [HttpPost("follow")]
+        public IActionResult Follow([FromBody]FollowersDto followersDto)
+        {
+
+            // "helped me alot" https://www.youtube.com/watch?v=j1e6Z-7QNpk
+
+            var followed = followersDto.FollowedUserId; // 
+            var following = followersDto.FollowerId;
+
+            var uuF = _context.Users.Find(followersDto.FollowedUserId);
```

Comment:
you should bring each if statements and place them immediately after the find statements

e.g.

```c#
    var followedUser = _context.Users.Find(followersDto.FollowedUserId);
    // check if the user to be followed exist in the database, or throw if it doesn't exist
    if (followedUser == null)
    {
         return BadRequest();
    }
```
Then after you'd do the same for the `followerUser` as well.

Reply:
ok